test/parallel_api/algorithm/alg.sorting/alg.set.operations/set.pass.cpp

```diff
@@ -281,14 +381,14 @@ main()
         int32_t;
 #endif
 
-    test_set<data_t, data_t>(oneapi::dpl::__internal::__pstl_less());
+    test_set<data_t, data_t>(oneapi::dpl::__internal::__pstl_less(),false);
```

Comment:
Tests below are not testing when `TEST_DPCPP_BACKEND_PRESENT==1`. Could you please add a test with `true`?

Reply:
Done

Reply:
Added